### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by bernardwang**
### Summary
- Bumped `design-system`, `design-system-docs`, `design-system-scripts` to 2.0.0
- Bumped `eslint-config-design-system` and `stylelint-config-design-system` to 2.0.0
- Updated example package.jsons
- Updated v2 banner copy

### Fixed
- Fixed an e2e test regarding copy

### CODE REVIEW

File: packages/design-system-docs/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "repository": "CMSgov/design-system",
   "license": "SEE LICENSE IN LICENSE.md",
   "dependencies": {
-    "@cmsgov/design-system": "2.0.0-beta.5",
+    "@cmsgov/design-system": "^2.0.0",
```
Comment (line47) [inline packages/design-system-docs/package.json:12]: Wondering why the `^` was used here and not with other dependencies?

File: design-assets/cms-design-system-ui-kit.xml [hunk 2]
```diff
@@ -9,9 +9,9 @@
     </image>
     <generator>Sketch</generator>
     <item>
-      <title>CMS Design System UI kit - v0.7</title>
-      <pubDate>Mon, 18 May 2020 15:11:00</pubDate>
-      <enclosure url="https://github.com/CMSgov/design-system/raw/master/design-assets/CMS-Design-System-UI-kit.sketch?raw=true" type="application/octet-stream" sparkle:version="0.7"/>
+      <title>CMS Design System UI kit - v2.0.0</title>
```
Comment (bernardwang) [inline design-assets/cms-design-system-ui-kit.xml:12]: @line47 does this look right for updating sketch? also is the library updated enough for v2?

Comment (bernardwang) [reply design-assets/cms-design-system-ui-kit.xml:12]: If there's updates to make we can hold off on updating sketch. I.e. what should we do with the focus styles in sketch?

Comment (line47) [reply design-assets/cms-design-system-ui-kit.xml:12]: That looks right! Thanks, @bernardwang I think we are good to update Sketch

File: packages/design-system-docs/package.json (hunk 1 above)
Comment (bernardwang) [reply packages/design-system-docs/package.json:11]: I think we don't want an exact version, this way when we update `design-system` we dont need to also update this dep. Not 100% sure though, we can adjust in the future if needed

**Pull request merged**